flinkprocessor/build.gradle

```diff
@@ -80,7 +79,7 @@ publishing {
                 username "$System.env.MAVEN_USERNAME"
```

Comment:
maybe mention in docs to set MAVEN_USERNAME and MAVEN_PASSWORD in system

Reply:
This is in README.md. In the next PR, this will be simpler.